app/mailers/notifier.rb

```diff
@@ -87,8 +87,12 @@ def statement_pdf
     @statement_pdf ||= StatementPdfFactory.instance(@statement)
   end
 
-  def send_nucore_mail(to, subject, template_name = nil)
-    mail(subject: subject, to: to, template_name: template_name)
+  def send_nucore_mail(to, subject, template_name = nil, reply_to = nil)
+    mail(subject: subject, to: to, template_name: template_name, reply_to: reply_to)
```

Comment:
If `reply_to` is `nil` it looks like we get a `Reply-To:` header with no value. GMail is smart enough to ignore it, but it would be nicer to not generate the header at all if it's blank.